index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books List</title>
+    <link rel="stylesheet" href="./bootstrap.min.css">
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
```

Comment:
The main page of your application should show one section of your application. Currently, your home page shows all the sections. Kindly fix this error.
![image](https://user-images.githubusercontent.com/33268927/201264590-5b66c5b2-3cb0-4436-8f5d-432a05044228.png)